setup.cfg

```diff
@@ -34,12 +34,11 @@ include_package_data = True
 setup_requires =
   setuptools_scm
 install_requires =
-  matplotlib>=3.3.0
-  ndcube>=2.0.1
-  sunpy>=3.0.0
-  mpl_animators
+  ndcube[plotting]>=2.0.1
```

Comment:
Should `numpy` and `astropy` be added since they are explicitly imported by sunraster?

Reply:
Probably.

Reply:
Added.